lib/manageiq/cross_repo/runner.rb

```diff
@@ -1,28 +1,29 @@
 require "manageiq/cross_repo/repository"
-require "active_support/core_ext/object/blank"
 
 module ManageIQ::CrossRepo
   class Runner
     attr_reader :test_repo, :core_repo, :gem_repos, :test_suite, :script_cmd
 
+
+
     def initialize(test_repo:, repos:, test_suite: nil, script_cmd: nil)
       @test_repo = Repository.new(test_repo || "ManageIQ/manageiq@master")
 
       core_repos, @gem_repos = Array(repos).collect { |repo| Repository.new(repo) }.partition(&:core?)
       @core_repo = core_repos.first
 
       if @test_repo.core?
-        raise ArgumentError, "You cannot pass a different core repo when running a core test" if @core_repo.present? && @core_repo != @test_repo
+        raise ArgumentError, "You cannot pass a different core repo when running a core test" if !@core_repo.nil? && @core_repo != @test_repo
```

Comment:
how about:

```suggestion
        raise ArgumentError, "You cannot pass a different core repo when running a core test" unless [@test_repo, nil].include?(@core_repo)
```

Reply:
Saves all of two characters, and makes the logic more confusing in my opinion.  Going to pass.